tests/acceptance/features/bootstrap/SharingNgContext.php

```diff
@@ -151,7 +152,7 @@ public function sendShareInvitation(string $user, TableNode $table): ResponseInt
 			$shareType = $shareTypes[$index];
 			$shareeIds[] = ($shareType === 'user')
 				? $this->featureContext->getAttributeOfCreatedUser($sharee, 'id')
-				: $this->featureContext->getAttributeOfCreatedGroup($sharee, 'id');
+				: ($this->featureContext->getAttributeOfCreatedGroup($sharee, 'id') ?: WebDavHelper::generateUUIDv4());
```

Comment:
i found using ternary operator here confusing. may be changing to if-else the whole thing would be readable. And also putting a comment why `WebDavHelper::generateUUIDv4()` is used. may be something like `for non-exixtent -group`